pepper-apis/src/main/resources/db-changes/schema/DDP-3709-file-upload.xml

```diff
@@ -23,16 +23,16 @@
             <column name="file_upload_guid" type="varchar(20)">
                 <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_guid_uk"/>
             </column>
-            <column name="blob_name" type="varchar(300)">
+            <column name="blob_name" type="varchar(1024)">
                 <constraints nullable="false"/>
             </column>
-            <column name="mime_type" type="varchar(50)">
+            <column name="mime_type" type="varchar(255)">
                 <constraints nullable="false"/>
             </column>
-            <column name="file_name" type="varchar(100)">
+            <column name="file_name" type="varchar(255)">
```

Comment:
While I'm in here, I'm bumping up `file_name` and `file_size` data types as well.